### DISCUSSION TITLE
Astro static site framework

### DISCUSSION THREAD

**Discussion by DanHugoDanHugo** [CodeForVegas/cfv-dotorg#1] [General]
Was chatting with @noliveira95 and he is aiming at [Astro](https://astro.build/blog/introducing-astro/) to build up at least the foundation website (the dotorg) and who knows, maybe the other dotvegas domains as well.

If our goal is to “eat our own dogfood,” then we should be able to access our own stuff via API calls or other means, which means building static pages server-side will be a useful check on our backend services, while the ability to include client-side calls to our own APIs will be a good check on the outside world trying out our stuff.

Seems like a good choice to move forward with.

**Comment by nickjolive** [comment]
I have been working on a new version of the site using Astro and Tailwind. I will most likely be updating the repo with the new codebase.

**Reply by DanHugoDanHugo** [reply]
Sounds good. If the calendar stuff is ready to try out for Monday (5 Sep) we can see how to integrate an embedded widget call. A nice trial by fire of the Astro approach (fwiw I am a big fan of server-side rendering either way…). Coolio.

**Reply by nickjolive** [reply]
I created a new repo. [Here is the link](https://github.com/CodeForVegas/cfv-dotorg). We will just need to transfer the discussions over, we can keep this one for ideation purposes or delete it, up to you.

**Reply by DanHugoDanHugo** [reply]
I enabled Discussions (like this we are having now) for the new repo you created, I am not certain if you would have been able to do that (should be able, in Settings for the repo). You will discover the pros and cons of starting with a brand new repo over forks or other means of staying in the same repo, though it would be much more fun if there were a bunch of collaborators to chime in…

**Reply by nickjolive** [reply]
Yeah, I thought about that but since it's so small with not many collaborators, I went with the easier option. Lol. In the future I will attempt to keep major changes in the same repository.
